cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -605,6 +615,10 @@ public TwillController start(long timeout, TimeUnit timeoutUnit) {
         }
       }
 
+      if (!isSystemNamespace(cdapRuntimeNamespace)) {
```

Comment:
should add a comment describing why this is only done if it's not the system namespace

Reply:
done